src/main/java/io/github/syst3ms/skriptparser/file/FileParser.java

```diff
@@ -84,4 +84,47 @@ private int count(List<FileElement> elements) {
         }
         return count;
     }
+
+    /**
+     * Removes all comments from a given String
+     * @param string the String
+     * @return the String with the comments removed; {@literal null} if no comments were found; an empty string if the whole String was a comment
+     */
+    @Nullable
+    private static String removeComments(String string) {
+        if (string.matches("^[\\s\\t]*#[^#]+") || string.equals("#") || string.isBlank()) {
+            return ""; // Whole string is a comment
+        }
+
+        var builder = new StringBuilder();
+        outer:
+        for (int i = 0; i < string.length(); i++) {
+            char c = string.charAt(i);
+
+            if (c == '#') {
+                if (string.charAt(i + 1) == '#') {
+                    builder.append(c).append(string.charAt(++i));
+                } else {
+                    var checked = string.substring(i + 1);
+                    for (int j : new int[] {3, 6, 8}) {
+                        if (checked.length() >= j
+                                && Color.COLOR_PATTERN.matcher(checked.substring(0, j)).matches()) {
+                            builder.append(c).append(checked, 0, j);
+                            i+=j;
+                            continue outer;
+                        }
+                    }
+
+                    // Comment was found. Erase it from the string
+                    assert !builder.toString().isEmpty();
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Use of identifier that is a keyword in later versions of Java (NM_FUTURE_KEYWORD_USED_AS_IDENTIFIER)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-use-of-identifier-that-is-a-keyword-in-later-versions-of-java-nm-future-keyword-used-as-identifier).
The identifier is a word that is reserved as a keyword in later versions of Java, and your code will need to be changed in order to compile it in later versions of Java.

Reply:
So you are saying 'builder' is a keyword in future Java versions? When? I can't seem to find any results with a simple Google search ('java builder keyword').
If I'm misunderstanding this, could you perhaps elaborate a bit more?
Thanks